test/specs/integration.spec.js

```diff
@@ -108,7 +110,7 @@ describe('Integration tests', function() {
       firepad1.setText('lorem ipsum');
       step(25);
     });
-  });
+  }, 6000000);
```

Comment:
Yup it was way overkill. I mostly wanted to see Travis pass then didn't come back and adjust. It is set to 5 mins now and is a variable so all extended timeouts are the same length.